medusa/network_timezones.py

```diff
@@ -116,7 +116,7 @@ def get_network_timezone(network, _network_dict):
     # Get the name of the networks timezone from _network_dict
     network_tz_name = _network_dict[network] if network in _network_dict else None
 
-    if network_tz_name is None and network not in missing_network_timezones:
+    if network_tz_name is None and network != '' and network not in missing_network_timezones:
```

Comment:
@ratoaq2 default value for network is ''. imo it should be None so we don't need to change this code